### DISCUSSION TITLE
Canadá ou Austrália? Diploma ead válido?

### DISCUSSION THREAD

**Discussion by jonathansc92** [frontendbr/forum#564] [Discussão]
Olá, tenho dúvida em qual país me encaixaria dou da área de Ti, técnico em informática, e estou cursando análise e desenvolvimento de sistemas, vejo que o Canadá tem os braços maus abertos para os brasileiros, posso estar errado e vejo a Austrália como  país que para nós é mais difícil as oportunidades. Diploma ead são válidos nesses países?

**Comment by cmpereirasi** [comment]
Bom dia @jonathansc92, isso vai depender do seu objetivo. Se você pretende emigrar talvez o Canada pode ser uma boa opção. Você pode aplicar para [Express-entry](http://www.cic.gc.ca/english/express-entry/) se tiver perfil para tal...
Se está pensando em sair pra estudar, talvez Austrália seria uma boa. Att

**Comment by jonathansc92** [comment]
Obrigado @cmpereirasi.
E enquanto ao diploma EAD no exterior é levado em consideração ou não?

**Comment by marcusvcribeiro** [comment]
No seu diploma nao diz se é EAD ou nao se eu nao me engano, para nao ter
essa descriminação.

2017-05-05 9:31 GMT-03:00 Jonathan Cruz <notifications@github.com>:

**Comment by jonathansc92** [comment]
@marcusvcribeiro Entendi, era só uma dúvida mesmo, pois aqui no Brasil já fiz entrevistas e o entrevistador me perguntou se era EAD que estava cursando, tenho impressão que há certos lugares que levam isto em consideração no Brasil.

**Comment by cmpereirasi** [comment]
@jonathansc92 pela minha concepção, quando se trata de TI o que vai impactar mais é a sua experiência na área, porém a diploma é levado em consideração ao aplicar para emigração. Mas faça uma pesquisa mais ao fundo. Uma dica não: Não deixe que a sua diploma em EAD seja uma barreira para realizar o seu sonho! Abraço

**Comment by jonathansc92** [comment]
@cmpereirasi Obrigado pela força, continuarei me aprofundando neste projeto. Abraço

**Comment by danilosilvadev** [comment]
Sim mas @jonathansc92 ainda tem uma questão. Muitos países só consideram nível superior possível de ser validado se tiver mais de 4 anos. Muitos cursos de TI no br são de 2 anos, 2,5 etc. Agora eu quero saber no CANADÁ explicitamente, aceitariam validar um curso de 2 anos como nível superior daqui do br? Essa é minha maior e grande dúvida pois tbm quero emigrar pra lá.

**Comment by matheussf** [comment]
@danilosilvadev  me pergunto se conseguiste sanar tua dúvida, pois tenho a mesma dúvida em relação a isto.

**Comment by danilosilvadev** [comment]
@matheussf bem, pelo visto não rola muito sem ser 4 anos mesmo. Mas se eu fizer agora uma facul de 4 anos no fim dela terei mais de 5 anos de XP, em torno de 7 a 8. O que confere um imigrante que não precisa de degree por causa do XP vc consegue outros tipos de vistos. Então, me pareceu melhor gastar o tempo q ficaria estudando assuntos da faculdade onde muitos n seriam úteis, em criar projetos, me desenvolver como pessoa e programador. Continuar estudando de td, inclusive coisas fora do escopo do coding como team manegement e leadership. Criar um portifólio legal, ser recomendado nas empresas que eu passar, juntar o dinheiro q gastaria numa mensalidade e ser feliz até lá. Sinceramente, considero meu tempo escasso e valioso demais pra n liderar meu desenvolvimento pessoal, então seguir o schedule de uma universidade n se adapta a minha personalidade. Mas outras pessoas podem fazer e gostar e se dar bem. Como fazer academia ou treinar em casa, se vc fizer direito ambos dão no mesmo resultado, e depende da personalidade do indivíduo pra saber qual seria o mais proveitoso.

**Comment by felipefialho** [comment]
Talvez o @RatoX possa ajudar aqui.

**Comment by andrecoliveira** [comment]
Aqui na Austrália o diploma é o requisito fundamental para qualquer tipo de visto e também de acesso para oportunidades na área. Em todos os processos que participei o diploma na área de TI era exigência, o que eu não sei dizer é a questão de ensinos EAD. Quanto a dificuldade para empregos, aqui com certeza é bem mais difícil que o Canadá, aqui 40% da população é estrangeira e, grande parte dela são os asiáticos. Em questão de programação, brigar por vagas com eles e os indianos é muito difícil, mas não impossível.

**Comment by brunooomelo** [comment]
Tecnólogo ou Bacharelado ?

**Comment by matheussf** [comment]
Bom, as minhas dúvidas são em relação a tecnólogo.

**Comment by danilosilvadev** [comment]
@matheussf tem um slack chamado https://brazil-tech-expats.slack.com onde tem brasileiros em vários países. Perguntei lá e isso que me responderam:

PP [Jan 7th at 4:17 PM]
Olá pessoal, tenho uma dúvida. Meu amigo disse que pra cada ano de universidade eu precisaria ter 3 anos de XP pra poder receber um VISA. Esquecendo o quão absurdo essa regra parece, é verdade isso no país em q vcs atuam? eu n pretendo ter degree mas eu tbm gostaria muito de sair do BR. Já trabalho na área, quantos anos de XP preciso ter?


35 replies
airtonzanon [3 days ago]
Isso depende muito do País e empresa mano


airtonzanon [3 days ago]
normalmente vejo a empresa não bancando o visto por que o cara não tem facul e/ou não passou no processo e não tendo problemas com o País (edited)


PP [3 days ago]
mas dá pra ir sem degree e sem 12 anos de XP?

airtonzanon [3 days ago]
Dá sim

airtonzanon [3 days ago]
aqui na Holanda acho que eles não pedem, porém dependendo da empresa vai pedir

PP [3 days ago]
Qual o mínimo de XP q na holanda precisa ter?


PP [3 days ago]
no caso, segundo o governo

bwagner [3 days ago]
pelo que eu entendo de Holanda, a parte boa é que não é o governo que define se tu é highly qualified, é a empresa que te contrata que é credenciada pelo governo pra fazer isso

bwagner [3 days ago]
já na Alemanha é o governo que decide se tu é highly qualified, aí diploma pesa mais, porque o governo não sabe julgar essas coisas

bwagner [3 days ago]
que eu saiba Holanda e Suécia tem esse esquema de deixar a decisão pras empresas pra contratar gente qualificada, talvez tenham outros países


gabrielreisn [3 days ago]
cara, um amigo meu conseguiu ir pra amsterda com 4-5 anos de xp sem diploma, imagino que eles esperam de vc uma qualificação pleno/senior


PP [3 days ago]
vlw gente, me sinto mais confiante. Netherlands é um bom destino

PP [3 days ago]
pra mim

gmmcal [3 days ago]
exatamente oq o @bwagner falou pra holanda… nao precisa d diploma, e esse lance d x anos por ano d faculdade me parece migué do teu amigo… com tanta demanda de dev, restringir por causa d diploma alguem q passou na selecao d uma empresa é quase burrice

guilopes [3 days ago]
Na Holanda não precisa comprovar nada fe experiencia, é puramente baseado no salario

guilopes [3 days ago]
Essa regra ai é pros USA

PP [3 days ago]
@gmmcal meu amigo é alemão, acho q ele se referiu ao país dele

gmmcal [3 days ago]
na alemanha precisa do diploma pra tirar o blue card. se for visto d trabalho normal e nao tiver diploma, basta comprovar (com cartas d recomendacao) a experiencia pra vaga… nao precisa d X anos por ano de faculdade

uaiHebert [3 days ago]
Moro na Alemanha, e nunca ouvi falar disso

bwagner [3 days ago]
Na Alemanha fui negado por experiência + de 7 anos, na resposta me disseram que não podiam avaliar o caso sem um diploma, então meio que depende da sorte

victorcampos [3 days ago]
na Alemanha é um saco porque se você não tem diploma precisa aplicar com um visto de "especialista", parece ser fácil conseguir (conheço uns 4 que estão em Berlin desse jeito)

victorcampos [3 days ago]
mas não lembro de precisar de N anos para converer a 1 de faculdade, isso eu lembro era regra pra H1B nos EUA quando fiz entrevista pro FB e LinkedIn

victorcampos [3 days ago]
aqui na Suécia é bem tranquilo, não tenho diploma e o governo não exigiu documentação alguma pra provar experiência, a empresa decide

victorcampos [3 days ago]
Holanda eu acredito que seja parecido porque um amigo meu mora lá e foi basicamente o mesmo processo que o meu, nenhuma exigência de experiência pelo governo

PP [3 days ago]
@victorcampos bom saber, entao vc só precisa encontrar um bom fit

Renan Chagas [3 days ago]
Acho que na Alemanha vai ser mais dificil vc conseguir, geralmente eles pedem diploma lá, a lei permite sem diploma porem é mais demora  e burocratico aí as empresas não costumam seguir. Tô a 6 meses fazendo processos, e só 2 empresas de Berlin toparam fazer o processo comigo sem diploma.

Holanda e Irlanda não ligam pra isso, mas mesmo assim algumas empresas exigem.

Meu foco tem sido nesses 3 países: Holanda, Irlanda e Alemanha. (edited)

Renan Chagas [3 days ago]
No meu processo da Amazon eles pediram 12 anos de experiencia comprovável para quem não tivesse graduação pra aplicar pra vaga em Berlin. Tive que desenterrar um estágio tecnico na epoca de colegio pra completar os 2 anos que faltavam e  não ser negado de cara.

Não sei se é regra da imigração de lá ou da empresa. (edited)

PP [3 days ago]
E canadá, alguém manja como é?


uaiHebert [3 days ago]
ja vi gente vir com menos de 12 anos. acho que eh algo da empresa viu

kinncj [3 days ago]
Canada: https://www.canada.ca/en/immigration-refugees-citizenship/services/work-canada/permit/temporary/eligibility.html
canada.ca
Find out if you’re eligible to work in Canada - Canada.ca
Eligibility requirements to apply for a work permit inside, outside and as you enter Canada

kinncj [3 days ago]
caso você queira imigrar: https://www.canada.ca/en/immigration-refugees-citizenship/services/come-canada-tool-immigration-express-entry.html
canada.ca
Do you want to come to Canada as a skilled immigrant? - Canada.ca
Answer a few questions to find out what immigration programs you can apply for.

kinncj [3 days ago]
no site do CIC vc encontra várias informações, até mesmo das mais diferentes formas de imigrar.

Thiago Fernandes [2 days ago]
Espanha não é o governo que decide. A decisão é da empresa. Se a empresa topar, já era.

lucasfcosta [2 days ago]
Aqui no Reino Unido também. Dá pra vir tranquilo sem diploma. Eles não pedem pra você comprovar anos de experiência, mas a empresa tem que provar que não conseguiu ninguém tão qualificado quanto você no mercado.

**Comment by thiagok** [comment]
@danilosilvadev Alguma chance de entrar neste canal? Estou planejando sair do país e algumas dicas seriam bem importantes. Obrigado.

**Comment by gmmcal** [comment]
@thiagok manda teu email q mando convite

**Comment by matheussf** [comment]
@gmmcal Podes me convidar também? Matheussf.sc@gmail.com
Muito obrigado

**Comment by gmmcal** [comment]
bom eh q o @danilosilvadev fez mention a uma galera q nao necessaramente tem o mesmo handle no github q no slack...

**Comment by thiagok** [comment]
@gmmcal klein.thiago@gmail.com valeu :+1:

**Comment by jonathansc92** [comment]
Galera, me informei mais sobre o assunto, nosso diploma vale desde que seja em alguma área reconhecido e que a tradução não fique zuada, mas vale ressaltar o diploma vale para quem quer entrar no Canadá através pelo processoFederal(EE), no caso do Canadá as empresas não ligam muito pra diploma como acontece aqui no Brasil, lá conta a forma que você se vende e as experiências profissionais, pra eles pouco interessa se você estudou na PUC, USP e etc, lá é outra realidade.

**Comment by RatoX** [comment]
Carai, vi essa notificação hoje.

As empresas aqui na Australia não se importam muito com diploma porém tu precisa de um para facilitar a burocracia do visto, eu não tinha diploma tive que correr atrás de uma carta assinada pelos gerentes dos meus ultimos 5 anos.

@jonathansc92 

Minha dica basica é: aguente sua raba no Brasil e termine sua faculdade, sua vida vai ser 100x melhor depois disso.

**Comment by luizfelipehh** [comment]
@RatoX 

E no meu caso que tenho graduação completa, estou fazendo pós, porém não tenho anos de experiência, as empresas na Austrália dão essa oportunidade ou querem caras Plenos/Senior?

**Comment by gmmcal** [comment]
@luizfelipehh não sei como é no Canadá ou Australia, mas sei que a Alemanha contrata entry-level people se tiverem um bom fit.

**Comment by RatoX** [comment]
@luizfelipehh burocraticamente falando é dificil tu justificar para o governo pq tá trazendo pessoas juniors de fora do pais, por isso as empresas focam em plenos/senior

**Comment by camillodev** [comment]
@RatoX  e @andrecoliveira  vcs sabem dizer se na australia conta o diploma de tecnologo ou somente bacharel? E pos lato sensu conta alguma coisa ou so mestrado? 
vlw :D

**Comment by andrecoliveira** [comment]
@camillodev A questão que eu cito sobre a exigência do diploma é mais utilizado na questão do visto. A empresa precisa comprovar que está contratando um profissional qualificado e tudo isso conta pontos lá na frente, caso busque um visto de residente ou até mesmo um sponsor. Se você pretende ir com o visto de estudante, a empresa não será tão exigente na questão do seu diploma. Mas caso fique definitivo, aí que entra toda essa burocracia. 

Caso tenha diploma de tecnólogo ou bacharel está ótimo e se tiver pós ou mestrado, é melhor ainda. Porque vai somando pontos para melhorar seu visto lá na frente. Tem alguns vídeos no youtube falando sobre os vistos, dê uma estudada.

Respondi sua dúvida?

**Comment by camillodev** [comment]
@andrecoliveira  respondeu sim cara. mto obrigado, Tava preocupado em relação ao visto permanente mesmo. Brigadao.

**Comment by PedroPini** [comment]
@RatoX saberia me informar se a carta assinada pelos antigos empregadores precisa ser de um gerente ou pode ser do RH mesmo?
